src/views/v2/commerce/checkout/cart.php

```diff
@@ -20,25 +20,24 @@
  * @var string           $paypal_attribution_id [Global] What is our PayPal Attribution ID.
  * @var int              $section               Which Section that we are going to render for this table.
  */
-use \TEC\Tickets\Commerce\Module;
 
 $post = get_post( $section );
 ?>
 
-<table>
-	<?php $this->template( 'checkout/table-header', [ 'section' => $section, 'post' => $post ] ); ?>
+<div class="tribe-tickets__commerce-checkout-cart">
+	<?php $this->template( 'checkout/cart/header', [ 'section' => $section, 'post' => $post ] ); ?>
 
-	<tbody>
-	<?php foreach ( $items as $item ) : ?>
-		<?php
-		if ( $item['obj']->get_event_id() !== $section ) {
-			continue;
-		}
-		?>
-		<?php $this->template( 'checkout/table-row', [ 'section' => $section, 'post' => $post, 'item' => $item ] ); ?>
-	<?php endforeach; ?>
-	</tbody>
+	<div class="tribe-tickets__commerce-checkout-cart-items">
+		<?php foreach ( $items as $item ) : ?>
+			<?php
+			if ( $item['event_id'] !== $section ) {
+				continue;
+			}
+			?>
+			<?php $this->template( 'checkout/cart/item', [ 'section' => $section, 'post' => $post, 'item' => $item ] ); ?>
+		<?php endforeach; ?>
+	</div>
 
-	<?php $this->template( 'checkout/table-footer', [ 'section' => $section, 'post' => $post ] ); ?>
-</table>
+	<?php $this->template( 'checkout/cart/footer', [ 'section' => $section, 'post' => $post ] ); ?>
```

Comment:
are these args needed if we are not using them in the template? or is it to provide theme developers these args?